### PR TITLE
ADSB: Updated mavlink msg

### PR DESCRIPTION
**Pull request by magicrub**
no more floats, all fixed point.
- convert altitude from float m/s to int32 mm/s (yes, I know.. don't ask)
- convert velocities from float m/s to uint16 cm/s (limited to about Mach 1.91)

### CODE REVIEW

**Comment by magicrub** [comment]
Mavlink PR https://github.com/mavlink/mavlink/pull/472

File: libraries/AP_ADSB/AP_ADSB.cpp [hunk 1]
```diff
@@ -149,7 +149,7 @@ void AP_ADSB::perform_threat_detection(void)
 Location AP_ADSB::get_location(const adsb_vehicle_t &vehicle) const
 {
     Location loc {};
-    loc.alt = vehicle.info.altitude * 100;
+    loc.alt = vehicle.info.altitude * 10; // convert mm to cm.
```
Comment (OXINARF) [inline libraries/AP_ADSB/AP_ADSB.cpp]: Comment says it is converting mm to cm but code is multiplying by 10. Should be divide by 10 right?

**Comment by OXINARF** [comment]
I made an inline comment, but it seems @tridge has already committed this?

**Comment by magicrub** [comment]
Hmm, yeah I think you're right. Thank you Francisco! PR here:
https://github.com/diydrones/ardupilot/pull/3320

**Comment by magicrub** [comment]
merged

**Pull request closed without merging**